### DISCUSSION TITLE
How to declare global var?

### DISCUSSION THREAD

**Discussion by devalexqt** [parcel-bundler/parcel#8075] [Q&A]
I need to pass `API_HOST` and `LANGUAGE` vars to my bundle, how to do that?

**Comment by mischnic** [comment]
The easiest way is using `process.env.API_HOST` and then setting that env variable when bundling `API_HOST="xyz" parcel build ...`

**Comment by devalexqt** [comment]
And how I can use it in js file?

**Reply by mischnic** [reply]
Like any other environment variable: https://parceljs.org/features/node-emulation/#environment-variables

**Comment by devalexqt** [comment]
Ok.
Is it possible to set env by dynamic route in development mode?
Fore example, i build multi language site like this: /en/index.html and /ua/index.html
